### PR TITLE
Election Info and Round/Ballot Count Fixes

### PR DESCRIPTION
**Pull request by dmzimmerman**
This adds submission of election info for the DoS dashboard. It also slightly refactors random seed submission (in that it renames the endpoint class to more accurately reflect what it does).

### CODE REVIEW

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * 
+ * @author Daniel M Zimmerman
+ * @version 0.0.1
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public class SetElectionInfo extends AbstractDoSDashboardEndpoint {
+  /**
+   * The "random seed" parameter.
+   */
+  public static final String RANDOM_SEED = "random_seed";
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java]: Copy-pasta?

Comment (dmzimmerman) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java]: I'm not certain what you are saying here...

**Comment by ranweiler** [comment]
Closes #537.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java (hunk 1 above)
Comment (ranweiler) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java]: Was this accidentally copy-pasted? It doesn't seem to be used anywhere in this class.

Comment (dmzimmerman) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SetElectionInfo.java]: Ah, now I see. I missed what class it was on.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java [hunk 2]
```diff
@@ -54,8 +54,8 @@ private ComparisonAuditController() {
   }
   
   /**
-   * Compute the ballot (cards) for audit, for a particular county dashboard, 
-   * and index range.
+   * Get the cast vote records to audit, in order, for the given county dashboard
+   * in the specified range in the audt sequence.
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java]: s/audt/audit

File: server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java [hunk 3]
```diff
@@ -315,10 +315,13 @@ public static boolean submitAuditCVR(final CountyDashboard the_dashboard,
     }
 
     updateCVRUnderAudit(the_dashboard);
+    final Round current_round = the_dashboard.currentRound();
```
Comment (ranweiler) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java]: Why did we move this here?

Comment (dmzimmerman) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java]: It made more sense to update it here than to update it in `updateCVRUnderAudit()`; the effect is the same.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java (hunk 2 above)
Comment (dmzimmerman) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java]: OK.

**Comment by nealmcb** [comment]
Note also, commits re "Fixed a number of issues related to counting ballots in rounds."

**Pull request merged**